Dockerfile.QA

```diff
@@ -197,11 +197,11 @@ RUN cp tritonbuild/identity/install/backends/identity/libtriton_identity.so \
     cp -r qa/L0_infer/. qa/L0_infer_cudashm && \
     mkdir -p qa/L0_infer_valgrind && \
     cp -r qa/L0_infer/. qa/L0_infer_valgrind && \
-    mkdir -p qa/L0_infer_tf2 && \
-    cp -r qa/L0_infer/. qa/L0_infer_tf2 && \
+    mkdir -p qa/L0_infer_tf1 && \
+    cp -r qa/L0_infer/. qa/L0_infer_tf1 && \
     if [ -d qa/L0_tf_image_models ]; then \
-        mkdir -p qa/L0_tf_image_models_tf2 && \
-            cp -r qa/L0_tf_image_models/. qa/L0_tf_image_models_tf2; \
+        mkdir -p qa/L0_tf_image_models_tf1 && \
+            cp -r qa/L0_tf_image_models/. qa/L0_tf_image_models_tf1; \
```

Comment:
where do these directories get used?

Reply:
These are for testing the specific TF version (TF1 in this case) on the CI: https://gitlab-master.nvidia.com/dl/dgx/tritonserver/-/blob/master/.gitlab-ci.yml#L2768

Reply:
I see the job for L0_infer_tf2 you referenced which makes sense with the old code IIUC - but wheres the job for L0_infer_tf1 then if you're chaning the dir names?

Reply:
Sorry I wasn't being clear before. I changed the name from L0_infer_tf2 to  L0_infer_tf1 in my MR: https://gitlab-master.nvidia.com/dl/dgx/tritonserver/-/merge_requests/620. These tests are only for testing the non-default TF version. Previous the default is TF1, so those tests are testing against TF2. Right now as we are move to TF2 as default, we only need to test for TF1 since L0_infer with TF2 is already tested in regular L0_infer test set.